packages/grid-pro/src/vaadin-grid-pro-edit-column-mixin.js

```diff
@@ -267,8 +270,16 @@ export const GridProEditColumnMixin = (superClass) =>
       this._setEditorValue(editor, get(this.path, model.item));
       editor._grid = this._grid;
 
-      this._focusEditor(editor);
-      requestAnimationFrame(() => this._focusEditor(editor));
+      const editInitiatorChar = this._editInitiatorChar;
+      this._editInitiatorChar = undefined;
+      requestAnimationFrame(() => {
+        if (editInitiatorChar) {
+          this._setEditorValue(editor, editInitiatorChar);
+          editor.focus();
```

Comment:
Should `_focusEditor` be used here as well?

Reply:
Using `_focusEditor` also selects the input text. If the editor is opened using a char, this should not be the case. Instead, it should only have focus. I extracted the text selection block to another method for clarity.

Reply:
Ah, I see. Thanks for the clarification.